src/main/jextract

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+DIR=`dirname $0`
+$DIR/runtime/bin/java -m org.openjdk.jextract/org.openjdk.jextract.JextractTool "$@"
```

Comment:
What do you think about adding a dependency on some shell variable (e.g. JEXTRACT_JAVA_OPTIONS) to pass extra arguments to jextract's JVM? I recall reaching for something like this e.g. to enable debug mode? (But, totally optional, more asking for an opinion)

Reply:
Yes, this seems like a good idea.

Previously I would just modify the launch script.